recipes/dd-opentracing-cpp/all/conanfile.py

```diff
@@ -42,8 +36,22 @@ def _compilers_minimum_version(self):
             "apple-clang": "7",
         }
 
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["BUILD_PLUGIN"] = False
+        tc.variables["BUILD_SHARED"] = self.options.shared
+        tc.variables["BUILD_STATIC"] = not self.options.shared
+        tc.variables["BUILD_TESTING"] = False
+        tc.generate()
+
+        deps = CMakeDeps(self)
+        deps.generate()
+
     def export_sources(self):
-        self.copy("CMakeLists.txt")
+        copy(self, "CMakeLists.txt", self.recipe_folder, self.export_sources_folder)
```

Comment:
That Cmake file is no longer needed and can be removed. 

```
if(CONAN_COMPILER MATCHES "clang" AND CONAN_COMPILER_VERSION VERSION_LESS "4.0")
    add_compile_options(-ftemplate-depth=1024)
endif()
```

This part should be passed to CMakeToolchain

Reply:
Removed support for clang lower than4 as this is really old anyway